src/libraries/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableArray.cs

```diff
@@ -81,6 +81,56 @@ public static ImmutableArray<T> Create<T>(T item1, T item2, T item3, T item4)
             return new ImmutableArray<T>(array);
         }
 
+        /// <summary>
+        /// Creates an <see cref="ImmutableArray{T}"/> with the specified elements.
+        /// </summary>
+        /// <typeparam name="T">The type of element stored in the array.</typeparam>
+        /// <param name="items">The elements to store in the array.</param>
+        /// <returns>An immutable array.</returns>
```

Comment:
```suggestion
        /// <returns>An immutable array containing the specified items.</returns>
```

Likewise for the the `<return/>` elements of the other Create methods.

Reply:
Fixed.